release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -45,6 +45,10 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-8-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-8-rhcos-butane"]
+==== Butane config transpiler improves processing of machine configs
+
+{product-title} now includes the Butane config transpiler to assist with producing and validating machine configs. Documentation now recommends using Butane to create machine configs for LUKS disk encryption, root disk RAID, and custom kernel modules.
```

Comment:
We are going to have some OCP facing docs for Butane, correct?

Should be link to them from this note?

Reply:
Docs call it "boot disk mirroring" rather than "root disk RAID", so we should probably use the same terminology here.

@miabbott We'll have some introductory material, and then Butane will be used in individual task-based docs.  We could link the introductory material here if desired.

Reply:
Yes, I will definitely come back and add the link once the Butane docs are live.